.travis.yml

```diff
@@ -9,12 +9,17 @@ python:
   - "2.7"
   - "3.5"
 
+env:
+  - TEST_DIR=root TEST_TYPE=lint
+  - TEST_DIR=root TEST_TYPE=unit
+  - TEST_DIR=utils
+
 install:
-  - pip install -r requirements.txt
```

Comment:
@rhcarvalho I'm open to tweaking it a bit more, but the bigger issue at play is that we have ansible in requirements.txt, but we will need the ability to template the version of ansible in tox. We aren't testing multiple versions of Ansible today, but we will definitely want to add that in the not too distant future, especially as the 2.3 release nears (maybe even before 2.2.2 releases).